lang/syn/src/lib.rs

```diff
@@ -369,6 +377,7 @@ pub struct CompositeField {
 // A type of an account field.
 #[derive(Debug, PartialEq)]
 pub enum Ty {
+    AccountsInfo,
```

Comment:
We should rename this `VecAccount` and allow one to use `Vec<Account<'info, MyAccount>>` along with `Vec<AccountInfo>`. Otherwise, it's more efficient for one to just use `ctx.remaining_accounts` directly.

Reply:
No problem, I'll get on it.